### PR TITLE
FieldPropsManager.hpp: forward TranCalculator

### PR DESCRIPTION
**Pull request by akva2**
Reduce hotness of TranCalculator.hpp by 80% (171 files, 42 files instead of 213)

### CODE REVIEW

File: opm/input/eclipse/EclipseState/Grid/FieldPropsManager.hpp [hunk 1]
```diff
@@ -31,6 +30,8 @@ class EclipseGrid;
 class Deck;
 class DeckKeyword;
 namespace Fieldprops {
+class TranCalculator;
+using TranMap = std::unordered_map<std::string, TranCalculator>;
```
Comment (bska) [inline opm/input/eclipse/EclipseState/Grid/FieldPropsManager.hpp:34]: Hm, this raises an interesting question.  We now have to declarations of `TranMap`&mdash;one here and one in [`TranCalculator.hpp`](https://github.com/OPM/opm-common/blob/a47716e97c36f119081d42983480df440b15538b/opm/input/eclipse/EclipseState/Grid/TranCalculator.hpp#L120).  The two declarations are equal right now, but how do we ensure that they remain equal?

Comment (bska) [reply opm/input/eclipse/EclipseState/Grid/FieldPropsManager.hpp:34]: >>  how do we ensure that [the two declarations of `TranMap`] remain equal?
>
> We don't really. My personal preference would be to drop the typedef,

Yeah, I think that'd be the best option here really.  As far as I can tell, there's nothing in the current master sources that **depends** on the `TranMap` name.  We can't know anything about out-of-tree uses, but we generally don't really care about those anyway...

Comment (akva2) [reply opm/input/eclipse/EclipseState/Grid/FieldPropsManager.hpp]: done.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  I'll merge into master.

**Pull request merged**